doc/source/ray-core/handling-dependencies.rst

```diff
@@ -402,6 +402,40 @@ Example:
   {"pip": ["torch", "ray[serve]"],
   "env_vars": {"A": "a", "B": "new", "C", "c"}}
 
+.. _runtime-env-faq:
+
+Frequently Asked Questions
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Are environments installed on every node?
+"""""""""""""""""""""""""""""""""""""""""
+
+If specifying your runtime environment for the job, in ``ray.init(runtime_env=...)``, then the environment will be installed on every node.  See :ref:`Per-Job <rte-per-job>` for more details.
+
+When is the environment installed?
+""""""""""""""""""""""""""""""""""
+
+When specified per-job, the environment is installed when you call ``ray.init()``.  
+When specified per-task or per-actor, the environment is called when the task is invoked or the actor is instantiated (i.e. when you call ``my_task.remote()`` or ``my_actor.remote()``.)
+See :ref:`Per-Job <rte-per-job>` :ref:`Per-Task/Actor, within a job <rte-per-task-actor>` for more details.
+
+Where are the environments cached?
+""""""""""""""""""""""""""""""""""
+
+Any local files downloaded by the envs are cached at ``/tmp/ray/session_latest/runtime_resources``.
```

Comment:
What happens if the cached environment is deleted? Will it be reinstalled when needed later on?

Reply:
```suggestion
Any local files downloaded by the environments are cached at ``/tmp/ray/session_latest/runtime_resources``.
```